src/teacher/ProgressBar.js

```diff
@@ -0,0 +1,74 @@
+import React, { useEffect, useState } from "react";
+import * as s from "./ProgressBar.sc";
+
+const ProgressBar = ({ student }) => {
```

Comment:
`StudentActivityBar` might be a better name for this?

Reply:
True that. ProgressBar is now in the exersises folder, StudentActivitybar is in the teacher folder.